src/wasm-lib/kcl/src/std/patterns.rs

```diff
@@ -285,8 +285,9 @@ pub async fn pattern_transform_2d(exec_state: &mut ExecState, args: Args) -> Res
 ///   |> patternTransform(3, transform, %)
 /// ```
 #[stdlib {
-     name = "patternTransform",
- }]
+    name = "patternTransform",
+    feature_tree_operation = true,
```

Comment:
Oo fun I have to figure out an icon

Reply:
Could you include `patternLinear3d` and `patternRadial3d` actually? I already have an icon for those anyway.